main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/beyondstorage/beyond-ftp/cmd"
+	"github.com/beyondstorage/beyond-ftp/pprof"
+)
+
+func main() {
+	pprof.StartPP()
```

Comment:
Maybe we can add a flag like `debug` to enable pprof? It is a performance waste here to always enable pprof.

It's OK for me to change this in another PR.

Reply:
sure, I will fix it in another PR